int16 id
int16 type
geometry_msgs/Point point
geometry_msgs/Point last_point
